src/main.py

```diff
@@ -1,222 +1,105 @@
-    elif col_type[key]['data_type'] == 'bytea':
-        if value is not None:
-            return str(value)
-        else:
-            if col_type[key]['is_nullable'] == 'NO':
-                return ""
-            else:
-                return None
-    else:
-        print("Cannot find mapping for type : " + col_type[key]['data_type'])
-        return None
-
-
-def disable_triggers(schema_name, table_name):
-    cur = destination.cursor(cursor_factory=psycopg2.extras.DictCursor)
-    sql = "ALTER TABLE %s.%s DISABLE TRIGGER ALL;" % (schema_name, table_name)
-    cur.execute(sql)
-    cur.close()
-
-
-def enable_triggers(schema_name, table_name):
-    cur = destination.cursor(cursor_factory=psycopg2.extras.DictCursor)
-    sql = "ALTER TABLE %s.%s ENABLE TRIGGER ALL;" % (schema_name, table_name)
-    cur.execute(sql)
-    cur.close()
+            sql_select = 'select ' + select_str + ' from %s.%s LIMIT %s OFFSET %s' \
+                         % (schema_name, table_name, cfg['source']['batch'], str(offset))
+        cur = source.cursor(cursor_factory=psycopg2.extras.DictCursor)
+        cur.execute(sql_select)
+        rows = cur.fetchall()
+        cur.close()
+        values_lst = list()
+        for row in rows:
+            values = ()
+            value_map = create_value_map(mapping, row)
+            for key in new_key_lst:
+                if key in value_map.keys():
+                    values = values + (evaluate_val(col_type_dest, key, value_map[key]),)
+                else:
+                    values = values + (evaluate_val(col_type_dest, key, None),)
+            values_lst.append(values)
+
+        dest = destination.cursor(cursor_factory=psycopg2.extras.DictCursor)
+        psycopg2.extras.execute_values(dest, sql + ' %s', values_lst, template=None, page_size=1000)
+        destination.commit()
+        dest.close()
 
 
 def main():
-    parser = argparse.ArgumentParser(description="Script for migrating data")
-    parser.add_argument('-s', help="Source schema name", required=True)
-    parser.add_argument('-t', help="Source table name", required=True)
-    parser.add_argument('-ds', help="Destination schema name", required=True)
-    parser.add_argument('-dt', help="Destination schema name", required=True)
-    arguments = parser.parse_args()
-    disable_triggers(arguments.ds, arguments.dt)
-    migrate(arguments.ds, arguments.dt, arguments.s, arguments.t)
-    enable_triggers(arguments.ds, arguments.dt)
+    scriptStartTime = datetime.now()
+    for table_name in src_dest_table_names:
+        logger.info("##########################################################################################")
+        logger.info('Migration Started For : ' + '%s' % table_name)
+        tableStartTime = datetime.now()
+        disable_triggers(dest_schema_name, table_name)
+        create_batch_insert(dest_schema_name, table_name, src_schema_name, table_name)
+        enable_triggers(dest_schema_name, table_name)
```

Comment:
Please abstract this logic in a separate wrapper script and call the methods rom the wrapper script. The purpose of this script should be just performing the migration.